src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -381,10 +381,6 @@ public bool Equals(TypeSymbolWithAnnotations other, TypeCompareKind comparison)
                             return false;
                         }
                     }
-                    else if ((comparison & TypeCompareKind.IgnoreInsignificantNullableModifiersDifference) == 0)
```

Comment:
>else if ((comparison & TypeCompareKind.IgnoreInsignificantNullableModifiersDifference) == 0) [](start = 20, length = 92)

Shouldn't we keep this block instead and adjust it assuming that the condition is always true (the bit is never set)?

Reply:
Indeed. Thanks
I think that both branches behaved the same though, since there's no insignificant differences anymore. So I don't think we can observe the difference.